presto-main/src/main/java/com/facebook/presto/server/QueryExecutionResource.java

```diff
@@ -121,7 +121,7 @@ public Response getTaskInfo(@PathParam("queryId") String queryId)
 
                 tasks.add(new Task(taskStatus.getTaskId().toString(),
                         taskStatus.getState().toString(),
-                        taskStatus.getSelf().getHost(),
+                        taskStatus.getSelf().getHost() + ":" + taskStatus.getSelf().getPort(),
```

Comment:
@electrum  looks like "query-execution.html" redraw() calls "/v1/query-execution/", which represents "QueryExecutionResource#getTaskInfo", in which "tasks" will add a "Task", the 3rd parameter of Task will be presented on "query-execution.html" page. But now, only host was displayed.